docs/examples/acme-cert.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       - cm-http-nginx.k8s.group
       - cm-http-nginx2.k8s.group
     - http01:
-        ingressName: my-gce-ingress
+        ingress: my-gce-ingress
```

Comment:
Hmm.. you are correct in changing this, but I actually think this field should probably be called `ingressName` anyway, given the other field that can be set in this stanza is `ingressClass`, and we are explicitly referencing an ingress by name (i.e. we use `secretName` in `certificate.spec`.

Thoughts? I need to dig out some other examples of us referencing resources by name so we are consistent.

Not a decision/should not affect this PR though!

Reply:
Yeah only changed it to fit types.go, but ingressName is better I think

Reply:
I'll leave it as `ingress` for this PR though just to be correct